woocommerce/woocommerce-gateway-stripe
# Fix some minor spacing issues introduced in #3449

## Changes proposed in this Pull Request:

I noticed while looking through the changes in #3449 that there were some white spacing issues that was leading to some misalignment. 

This PR fixes those.

## Testing instructions

This PR just includes whitespace changes and code review should be enough. 

---


**Post merge**
